### PR TITLE
New feature - sketch on sheet metal

### PR DESCRIPTION
**Pull request by jaisekjames**
sketch on sheet metal
* Select a face & select a sketch.
It will create cut based on sketch.

mobile stand 1
mobile stand 2

### CODE REVIEW

**Comment by luzpaz** [comment]
1. Patched Sheetmetal workbench with new code 
    ```
    cd ~/.FreeCAD/Mod/sheetmetal/
    git fetch 
    curl https://patch-diff.githubusercontent.com/raw/shaise/FreeCAD_SheetMetal/pull/146.patch | git apply
    ```
    Resulted in:
    ```
    patching file SketchOnSheetMetalCmd.py
    patching file InitGui.py
    patching file SketchOnSheet.svg
    ```
2. Fired up FC
3. Switched to sheetmetal wb
Result:
    ```
   05:47:20  name 'SketchOnSheetMetalCmd' is not defined
   05:47:20  Traceback (most recent call last):
    File "<string>", line 61, in Initialize
   ```
```
OS: Manjaro Linux (KDE//usr/share/xsessions/plasma)
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 0.19.23074 (Git) AppImage
Build type: Release
Branch: master
Hash: 74b866ed3022fd1452c534dae31a721bcc668f2f
Python version: 3.8.6
Qt version: 5.12.5
Coin version: 4.0.0
OCC version: 7.4.0
Locale: English/United States (en_US)
```

File: InitGui.py [hunk 1]
```diff
@@ -58,8 +58,9 @@ def Initialize(self):
         import SheetMetalRelief
         import SheetMetalJunction
         import SheetMetalBend
+        SketchOnSheetMetalCmd
```
Comment (luzpaz) [inline InitGui.py]: doesn't this need to be `import SketchOnSheetMetalCmd`  
if so then I've submitted a PR https://github.com/jaisekjames/FreeCAD_SheetMetal/pull/1

Comment (jaisekjames) [reply InitGui.py]: Thank you.

Comment (luzpaz) [reply InitGui.py]: np, but it still doesn't fix the issue :disappointed:

Comment (jaisekjames) [reply InitGui.py]: I just downloaded my "new feature" branch zip & pasted on mod folder. It is working. No error.

Comment (luzpaz) [reply InitGui.py]: my bad. I wasn't applying the patch correctly. I needed to use `git apply` (i've amended the steps above (https://github.com/shaise/FreeCAD_SheetMetal/pull/146#issuecomment-735216604) to reflect that

**Comment by luzpaz** [comment]
@jaisekjames BTW, when I added the patch I get this output:
```
<stdin>:25: trailing whitespace.
#  
<stdin>:27: trailing whitespace.
#  
<stdin>:32: trailing whitespace.
#  
<stdin>:37: trailing whitespace.
#  
<stdin>:42: trailing whitespace.
#  
warning: squelched 36 whitespace errors
warning: 41 lines add whitespace errors.
```
Do you mind cleaning up the trailing whitespace?

**Comment by shaise** [comment]
Thanks for this great feature!

**Pull request merged**